script/utils/update-content-versions.ts

```diff
@@ -0,0 +1,50 @@
+import fs from 'fs';
+import path from 'path';
+import { makeUnifiedBookLoader } from '../../src/app/content/utils';
+import { ARCHIVE_URL, REACT_APP_ARCHIVE_URL, REACT_APP_OS_WEB_API_URL } from '../../src/config';
+import books from '../../src/config.books';
+import createArchiveLoader from '../../src/gateways/createArchiveLoader';
+import createOSWebLoader from '../../src/gateways/createOSWebLoader';
+
+export interface SimpleBook {
+  bookId: string;
+  versionNumber: string;
+}
+
+const booksPath = path.resolve(__dirname, '../../src/config.books.json');
+
+const bookLoader = (newArchiveUrl: string) => makeUnifiedBookLoader(
+  createArchiveLoader(newArchiveUrl, {
+    archivePrefix: ARCHIVE_URL,
+  }),
+  createOSWebLoader(`${ARCHIVE_URL}${REACT_APP_OS_WEB_API_URL}`)
+);
```

Comment:
i think you were going to move this and use the newBookLoader from the other file passed in right? maybe you just haven't gotten to that change yet